### ISSUE TITLE
How to know which is the oldest version of ppxlib for each version of the AST ?

### ISSUE THREAD

**Issue by Drup** [ocaml-ppx/ppxlib#243]
It seems like that should be part of the documentation somewhere, but it doesn't look like it. In particular, I would like to answer the simple question: "What's the oldest version of ppxlib for the AST version I'm currently using ?"

**Comment by pitag-ha** [comment]
Yes, that information should indeed be easy to find. Given an AST, I understand that you would like to find the oldest version of ppxlib that supports all its syntax. Is that right?

One way to find that out is to have a look at the release notes: you have to look for the entry "Bump AST to <the version you're interested in>". For the 4.12 AST that's ppxlib 0.22.0 (see [release notes](https://github.com/ocaml-ppx/ppxlib/releases/tag/0.22.0)); for the 4.11 AST it's ppxlib 0.18.0 (see [release notes](https://github.com/ocaml-ppx/ppxlib/releases/tag/0.18.0)) and so on. Would you expect to find that information somewhere else apart from the release notes? Where would you expect to find it?

**Comment by Drup** [comment]
> Where would you expect to find it?

In the readme, with a small table: AST version/ppxlib version(s).

**Comment by pitag-ha** [comment]
Ok, that sounds good to me. I'll come back to this when I get to improve the documentation. Thanks!

By the way, also have a look at [this](https://github.com/ocaml-ppx/ppxlib/issues/232#issuecomment-856779978) . That's not an informative approach as you're proposing, but it would be a means to restrict to the `ppxlib` versions with the AST you need. If we get to implement that, would that also meet your purpose?
